intermediate_source/dist_pipeline_parallel_tutorial.rst

```diff
@@ -0,0 +1,366 @@
+Distributed Pipeline Parallelism Using RPC
+==========================================
+**Author**: `Shen Li <https://mrshenli.github.io/>`_
+
+Prerequisites:
+
+-  `Single-Machine Model Parallel Best Practices <https://pytorch.org/tutorials/intermediate/model_parallel_tutorial.html>`__
+-  `Getting started with Distributed RPC Framework <https://pytorch.org/tutorials/intermediate/rpc_tutorial.html>`__
+-  RRef helper functions:
+   `RRef.rpc_sync() <https://pytorch.org/docs/master/rpc.html#torch.distributed.rpc.RRef.rpc_sync>`__,
+   `RRef.rpc_async() <https://pytorch.org/docs/master/rpc.html#torch.distributed.rpc.RRef.rpc_async>`__, and
+   `RRef.remote() <https://pytorch.org/docs/master/rpc.html#torch.distributed.rpc.RRef.remote>`__
+
+
+This tutorial uses a Resnet50 model to demonstrate implementing distributed
+pipeline parallelism with `torch.distributed.rpc <https://pytorch.org/docs/master/rpc.html>`__
+APIs. This can be viewed as the distributed counterpart of the multi-GPU
+pipeline parallelism discussed in
+`Single-Machine Model Parallel Best Practices <model_parallel_tutorial.html>`_.
+
+.. note:: This tutorial requires PyTorch v1.6.0 or above.
+
+Basics
```

Comment:
Suggestion: Add a link to the related example - pytorch/examples#749 - either at the top of this tutorial or at the end in a "Learn More" section.